src/harness/testcases/WINNF_FT_S_PCR_testcase.py

```diff
@@ -342,3 +423,629 @@ def test_WINNF_FT_S_PCR_1(self, config_filename):
+    for pal_record in config['palRecords']:
+      self._sas_admin.InjectPalDatabaseRecord(pal_record)
+
+    # Trigger SAS UUT to create a PPA boundary.
+    pal_ids = [record['palId'] for record in config['palRecords']]
+    ppa_creation_request = {
+        "cbsdIds": cbsd_ids,
+        "palIds": pal_ids
+    }
+
+    # Trigger PPA Creation to SAS UUT.
+    ppa_id = self.triggerPpaCreationAndWaitUntilComplete(ppa_creation_request)
+    logging.debug('ppa_id received from SAS UUT:%s', ppa_id)
+
+    # Trigger Full Activity Dump and retrieves the PPA Zone record.
+    uut_ppa_zone_data = self.triggerFadGenerationAndRetrievePpaZone(
+        ssl_cert=config['sasTestHarnessCert'],
+        ssl_key=config['sasTestHarnessKey'])
+
+    # Check if the PPA generated by the SAS UUT is fully contained within the service area.
+    logging.debug("SAS UUT PPA - retrieved through FAD:%s",
+                  json.dumps(uut_ppa_zone_data, indent=2, sort_keys=False,
+                             separators=(',', ': ')))
+    logging.debug("Reference model PPA - retrieved through PpaCreationModel:%s",
+                  json.dumps(json.loads(test_harness_ppa_geometry), indent=2, sort_keys=False,
+                             separators=(',', ': ')))
+
+    uut_ppa_geometry = uut_ppa_zone_data['zone']['features'][0]['geometry']
+    self.assertTrue(isPpaWithinServiceArea(config['palRecords'], uut_ppa_geometry),
+                    msg="PPA Zone is not within service area")
+
+    # Check the Equation 8.3.1 in Test Specfification is satisified w.r t
+    # [n.12, R2-PAL-05]. Check the area of the non-overlapping difference between
+    # the maximum PPA boundary created by SAS UUT shall be no more than 10% of the area
+    # of the maximum PPA boundary created by the Reference Model.
+    self.assertTrue(utils.PolygonsAlmostEqual(test_harness_ppa_geometry,
+                                              uut_ppa_geometry))
+
+  def generate_PCR_3_default_config(self, filename):
+    """Generate the WinnForum configuration for PCR 3."""
+    # File path where SAS UUT claimed ppa boundary generated in PCR.1 test.
+    pcr_1_test_config_file_path = os.path.join('testcases', 'configs',
+                                               'test_WINNF_FT_S_PCR_1',
+                                               'default.config')
+    sas_uut_claimed_ppa_boundary_file_path = getSasUutClaimedPpaBoundaryFilePath(
+        'default.config')
+
+    # Load SAS UUT claimed ppa boundary and check if any error while retrieving
+    # SAS UUT claimed ppa boundary generated in PCR.1 test.
+    try:
+      with open(sas_uut_claimed_ppa_boundary_file_path, 'r') as claimed_ppa_file:
+        user_claimed_ppa_contour = json.load(claimed_ppa_file)
+    except IOError:
+      raise RuntimeError('ConfigError:There is an error in reading path:%s \n\n'
+                         % sas_uut_claimed_ppa_boundary_file_path)
+
+    # Shrink the user claimed ppa boundary by 1 kilometer.
```

Comment:
more correct to say:
 " by approximately 1 kilometer"
Same in other place of this PR.

0.01degree is 1.1km in the latitude direction, but dependent on the overall latitude in the longitude direction.

Reply:
Thanks.   Comment is updated accordingly